### ISSUE TITLE
gRPC Building Error

### ISSUE THREAD

**Issue by eblanchette** [temporalio/ui#1717]
I can no longer build the project. This error started happening since 2.17.1

```
Compiling gRPC...
rm -rf api
mkdir -p api
protoc -I proto/api -I ./proto/dependencies/github.com/grpc-ecosystem/grpc-gateway/ -I ./proto/dependencies/github.com/gogo/googleapis/ -I ./proto/dependencies/api/ -I ./proto/dependencies/ --gogoslick_out=plugins=grpc,paths=source_relative,Mgoogle/protobuf/wrappers.proto=github.com/gogo/protobuf/types,Mgoogle/protobuf/duration.proto=github.com/gogo/protobuf/types,Mgoogle/protobuf/empty.proto=github.com/gogo/protobuf/types,Mgoogle/protobuf/descriptor.proto=github.com/gogo/protobuf/protoc-gen-gogo/descriptor,Mgoogle/protobuf/timestamp.proto=github.com/gogo/protobuf/types,Mgoogle/api/annotations.proto=github.com/gogo/googleapis/google/api:api --grpc-gateway_out=allow_patch_feature=false,paths=source_relative:api --openapiv2_out=openapi/assets proto/api/temporal/api/operatorservice/v1/*.proto ;  protoc -I proto/api -I ./proto/dependencies/github.com/grpc-ecosystem/grpc-gateway/ -I ./proto/dependencies/github.com/gogo/googleapis/ -I ./proto/dependencies/api/ -I ./proto/dependencies/ --gogoslick_out=plugins=grpc,paths=source_relative,Mgoogle/protobuf/wrappers.proto=github.com/gogo/protobuf/types,Mgoogle/protobuf/duration.proto=github.com/gogo/protobuf/types,Mgoogle/protobuf/empty.proto=github.com/gogo/protobuf/types,Mgoogle/protobuf/descriptor.proto=github.com/gogo/protobuf/protoc-gen-gogo/descriptor,Mgoogle/protobuf/timestamp.proto=github.com/gogo/protobuf/types,Mgoogle/api/annotations.proto=github.com/gogo/googleapis/google/api:api --grpc-gateway_out=allow_patch_feature=false,paths=source_relative:api --openapiv2_out=openapi/assets proto/api/temporal/api/workflowservice/v1/*.proto ;
temporal/api/workflowservice/v1/service.proto:425:43: "UpdateWorkerBuildIdCompatibilityRequest" is not defined.
temporal/api/workflowservice/v1/service.proto:425:93: "UpdateWorkerBuildIdCompatibilityResponse" is not defined.
temporal/api/workflowservice/v1/service.proto:427:40: "GetWorkerBuildIdCompatibilityRequest" is not defined.
temporal/api/workflowservice/v1/service.proto:427:87: "GetWorkerBuildIdCompatibilityResponse" is not defined.
temporal/api/workflowservice/v1/service.proto:445:36: "GetWorkerTaskReachabilityRequest" is not defined.
temporal/api/workflowservice/v1/service.proto:445:79: "GetWorkerTaskReachabilityResponse" is not defined.
make[1]: *** [build-grpc] Error 1
make: *** [build-ui-server] Error 2
```

**Comment by tdeebswihart** [comment]
I've recreated that with a clean checkout of v2.17.1. This is out of my wheelhouse so I'm going to pass it off to the team that manages this

**Comment by tdeebswihart** [comment]
I've confirmed we have the same issue on a fresh checkout of v2.20.0.

This looks like we've somehow mixed gogo/protobuf and google/protobuf interfaces together. This will go away when we bury gogo/protobuf, but we can likely fix it faster:

```
$ make build                                                
Compiling gRPC...
Fixing gRPC output paths
go mod tidy
go build -o ui-server ./cmd/server/main.go
# github.com/temporalio/ui-server/v2/api/operatorservice/v1
api/operatorservice/v1/service.pb.gw.go:57:9: cannot use msg (variable of type *"go.temporal.io/api/operatorservice/v1".ListSearchAttributesResponse) as protoreflect.ProtoMessage value in return statement: *"go.temporal.io/api/operatorservice/v1".ListSearchAttributesResponse does not implement protoreflect.ProtoMessage (missing method ProtoReflect)
```

**Comment by Alex-Tideman** [comment]
Should be fixed with 2.21.0

**Comment by eblanchette** [comment]
Confirmed!  Project builds successfully.  Thanks
